src/controllers/session/login/index.html

```diff
@@ -9,7 +9,10 @@ <h1 class="sectionTitle">${HeaderPleaseSignIn}</h1>
             </div>
 
             <div class="inputContainer">
-                <input is="emby-input" id="txtManualPassword" type="password" label="${LabelPassword}" autocomplete="current-password" />
+                <input is="emby-input" id="txtManualPassword" type="password" label="${LabelPassword}" autocomplete="current-password">
+                <button is="paper-icon-button-light" class="passwordToggle" type="button">
```

Comment:
You can remove `type="button"`.
```suggestion
                <button is="paper-icon-button-light" class="passwordToggle">
```